cmd/thanos/query.go

```diff
@@ -536,3 +550,36 @@ func firstDuplicate(ss []string) string {
 
 	return ""
 }
+
+func engineFunc(newEngine func(time.Duration) *promql.Engine, lookbackDelta time.Duration, dynamicLookbackDelta bool) func(int64) *promql.Engine {
+	deltas := []int64{downsample.ResLevel0}
+	if dynamicLookbackDelta {
+		deltas = []int64{downsample.ResLevel0, downsample.ResLevel1, downsample.ResLevel2}
+	}
+	var (
+		engines       = make([]*promql.Engine, len(deltas))
+		defaultEngine = newEngine(lookbackDelta)
```

Comment:
Default? Why not just raw? resolution 0 is raw, so this should be default one, no? :thinking:

Reply:
yeah, that does look better